spacify-lib/src/main/java/org/uci/spacifyLib/entity/AvailableSlotsEntity.java

```diff
@@ -0,0 +1,37 @@
+package org.uci.spacifyLib.entity;
+
+import com.sun.istack.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import java.sql.Time;
+
+import static org.uci.spacifyLib.utilities.Constants.SCHEMA_NAME;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "available_slots", schema = SCHEMA_NAME)
+public class AvailableSlotsEntity {
+    @Id
+    @NotNull
+    @Column(name = "available_slots_id")
+    private Long availableSlotsId;
+
+    @Column(name = "time_from")
+    private Time timeFrom;
+
+    @Column(name = "time_to")
+    private Time timeTo;
+
+    @Column(name = "room_type")
+    private String roomType;
```

Comment:
Keep this as RoomType ENUM like we have done for room entity. So that we have consistency across the database.

Reply:
Like I mentioned on WhatsApp, please add the table creation query etc to the sql file so that we can replicate this table in our local databases as well.